spec/index.bs

```diff
@@ -1655,6 +1661,7 @@ interface GPUSupportedLimits {
     readonly attribute unsigned long minUniformBufferOffsetAlignment;
     readonly attribute unsigned long minStorageBufferOffsetAlignment;
     readonly attribute unsigned long maxVertexBuffers;
+    readonly attribute unsigned long long maxBufferSize;
```

Comment:
Is there a way to say that this should always be at least as big as `maxUniformBufferBindingSize` and `maxStorageBufferBindingSize`?

Reply:
#3380